packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -1336,7 +1347,19 @@ module.exports.external = [
         return "";
       }
       const match = str.match(new RegExp(`^(${DECIMAL_REGEX_STR})$`));
-      return match ? match[0] : "";
+
+      if (match) {
+        const decimal = match[0];
+        const leadsWithDecimal = decimal.startsWith(".");
+
+        if (leadsWithDecimal) {
+          return `0${decimal}`;
```

Comment:
If we dont add a zero in front of a leading decimal it breaks the structure of the json